config/main.go

```diff
@@ -1,14 +1,21 @@
 package config
```

Comment:
Is this being maintained somewhere? Apologies if I missed it.
Snippet from issue:
> maintain this as a list in the code so that we can define the order in which we will check the presence of the conf file.

Reply:
How's this handled?
> The default path of the dice.conf file should be /etc/dicedb/dice.conf. "But by passing -o in init-config we should be able to change the location."

Reply:
-o flag will be implemented by today

Reply:
-o command is supported now